js/setup.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var WIZARD_NAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var WIZARD_SURNAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var WIZARD_COAT_COLORS = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var WIZARD_EYES_COLORS = ['black', 'red', 'blue', 'yellow', 'green'];
+
+var similarList = document.querySelector('.setup-similar-list');
+var template = document.querySelector('#similar-wizard-template').content.querySelector('.setup-similar-item');
+
+var wizards = [];
+
+// получение случайного индекса элемента
+var getRandomIndex = function (min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min)) + min;
+};
+
+// добавляет объекты магов в массив
+var pushObjects = function (array) {
+  for (var i = 0; i < 4; i++) {
+    var wizardNew = {
+      name: WIZARD_NAMES[getRandomIndex(0, WIZARD_NAMES.length)],
+      coatColor: WIZARD_COAT_COLORS[getRandomIndex(0, WIZARD_COAT_COLORS.length)],
```

Comment:
```suggestion
      coatColor: function,
```
посмотри, у тебя этот код делает одно и тоже, у него только входные данные меняются, предлагаю вынести его в функцию